diff --git a/index.js b/index.js
index 707daa0ea3e6bf264bf55d2b3726e33ebf6a529f..34e21d01ffe99a6e42a30ef846fe8f9c17e59d8a 100644
--- a/index.js
+++ b/index.js
@@ -278,7 +278,7 @@ function configure (options) {
         return undefined
       case 'bigint':
         if (bigint) {
-          return String(value)
+          return '"' + String(value) + 'n"'
         }
         // fallthrough
       default:
@@ -367,7 +367,7 @@ function configure (options) {
         return undefined
       case 'bigint':
         if (bigint) {
-          return String(value)
+          return '"' + String(value) + 'n"'
         }
         // fallthrough
       default:
@@ -477,7 +477,7 @@ function configure (options) {
         return undefined
       case 'bigint':
         if (bigint) {
-          return String(value)
+          return '"' + String(value) + 'n"'
         }
         // fallthrough
       default:
@@ -577,7 +577,7 @@ function configure (options) {
         return undefined
       case 'bigint':
         if (bigint) {
-          return String(value)
+          return '"' + String(value) + 'n"'
         }
         // fallthrough
       default:
