diff --git a/lib/commonjs/wallet.js b/lib/commonjs/wallet.js
index f7d8a67177bee9c3e2a3f4049ed575a25b477b9d..d15a22ef826a7db701b14ba3abf313bd7de44721 100644
--- a/lib/commonjs/wallet.js
+++ b/lib/commonjs/wallet.js
@@ -167,7 +167,8 @@ class Wallet extends web3_types_1.Web3BaseWallet {
         let index = this.length;
         const existAccount = this.get(account.address);
         if (existAccount) {
-            console.warn(`Account ${account.address.toLowerCase()} already exists.`);
+            // PATCH: disable this annoying warning
+            // console.warn(`Account ${account.address.toLowerCase()} already exists.`);
             index = (_a = this._addressMap.get(account.address.toLowerCase())) !== null && _a !== void 0 ? _a : index;
         }
         this._addressMap.set(account.address.toLowerCase(), index);
diff --git a/lib/esm/wallet.js b/lib/esm/wallet.js
index 74a80ed19b871e6eb8115846c56df56065e997f1..48e724c7b99eae2f681e5e12e0021472a33ec2a9 100644
--- a/lib/esm/wallet.js
+++ b/lib/esm/wallet.js
@@ -164,7 +164,8 @@ export class Wallet extends Web3BaseWallet {
         let index = this.length;
         const existAccount = this.get(account.address);
         if (existAccount) {
-            console.warn(`Account ${account.address.toLowerCase()} already exists.`);
+            // PATCH: disable this annoying warning
+            // console.warn(`Account ${account.address.toLowerCase()} already exists.`);
             index = (_a = this._addressMap.get(account.address.toLowerCase())) !== null && _a !== void 0 ? _a : index;
         }
         this._addressMap.set(account.address.toLowerCase(), index);
